css-map.json

```diff
@@ -2045,5 +2045,7 @@
     "NiCdLCpp3o2z6nBrayOn": "lyrics-lyricsContent-lyric",
     "MEjuIn9iTBQbnCqHpkoQ": "lyrics-lyricsContent-highlight",
     "kGR_hu4tdj9PnUlSPaRL": "lyrics-lyricsContent-provider",
-    "arY01KDGhWNgzlAHlhpd": "lyrics-lyricsContent-active"
+    "arY01KDGhWNgzlAHlhpd": "lyrics-lyricsContent-active",
+    "HxblHEsl2WX2yhubfVIc": "lyrics-lyricsContent-unsynced",
+    "KDhLFoEqoClhH12bsfrS": "lyrics-lyricsContent-description"
```

Comment:
This class isn't found anywhere on the latest version of Spotify (`1.2.0`), where exactly did you find this?
![image](https://user-images.githubusercontent.com/77577746/205629459-998e49b1-33d4-44e9-9dcd-545b893d5d3c.png)

Reply:
in 1.2.0 they reverted that "KDhLFoEqoClhH12bsfrS" (1.1.99) class to old "iq4cgi0YEKr6DGaTtzUj" (1.1.98)